.gitignore

```diff
@@ -14,13 +17,159 @@ node_modules
 
 # misc
 .DS_Store
+*.pem
+
+# local env files
+.env*.local
+.env
 .env.local
 .env.development.local
 .env.test.local
 .env.production.local
 
+# next.js
+/.next/
+/out/
+
+# Logs
+logs
+*.log
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+lerna-debug.log*
+.pnpm-debug.log*
+
+# vercel
 .vercel
+
+# typescript cache
+*.tsbuildinfo
+
+# typeScript declaration
+next-env.d.ts
+
```

Comment:
There is no need to add anything related to Typescript as of now. Please remove these.